projects/cashmere/src/lib/input/hc-form-field.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="hc-form-field-wrapper">
+    <span class="hc-form-field-label-wrapper">
+                <label class="hc-form-field-label"
+                       [attr.for]="_control.id">
+                    <ng-content select="hc-label"></ng-content>
+                    <span class="hc-required-marker" aria-hidden="true"
+                          *ngIf="(_control.required )&& !_control.disabled">&nbsp;*</span>
```

Comment:
in this scenario `_control` is public in the sense that it is visible to the template, but it's marked with an underscore meaning it's private in the sense that it's not part of the `hc-form-field`'s public api. Is this a correct understanding?

Reply:
That's correct. We'll have to adopt some kind of convention to be able to distinguish public API vs private and underscore is very common